### DISCUSSION TITLE
Helping a Newbie

### DISCUSSION THREAD

**Discussion by thefixxxer9211** [DIGITALCRIMINALS/OnlyFans#1061] [General]
So I have the latest Python & Poetry installed (running Windows 10) and figured out how to add those programs to PATH and that's where I'm stuck. I had the last official release (7.6.1 I think?) working but haven't been able to get anything going since. Any assistance is appreciated.

![scraper](https://github.com/DIGITALCRIMINALS/UltimaScraper/assets/141587914/f0c4db37-39a2-43ef-80fc-b743ae771bcd)

**Comment by babymasterr** [comment]
Run the following command to install 'requests' module:
`pip install requests`

You should now be able to run the updater and starts_us scripts

**Reply by thefixxxer9211** [reply]
OK, I did as you stated, now I get this: "Fatal error in launcher: Unable to create process using 'C:\...\python.exe, C:\...\pip.exe' install requests. The system cannot find the file specified"

**Reply by passeee** [reply]
@thefixxxer9211 
quick tut
install Python 3.10.1
install Poetry

start cmd in the UltimaScraper Folder
pip install -r requirements.txt

use this command to use "poetry in cmd":
set PATH=%PATH%;C:\Users\<username>\AppData\Roaming\pypoetry\venv\Scripts
use this command to tell poetry which python version should be used:
poetry env use "path tho python 3.10.1/python.exe"

pip install requests
python updater.py
poetry run python start_us.py

**Reply by thefixxxer9211** [reply]
I entered the pip install -r requirements.txt command as instructed & received the same error as I posted above.

**Reply by passeee** [reply]
This error is common you can search in Google for it.
Either reinstall Python or try python -m pip install --upgrade pip if that doesn't work then do other researching
